CollAction/Helpers/ProjectService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using CollAction.Data;
+using CollAction.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+
+namespace CollAction.Helpers
+{
+    public class ProjectService
```

Comment:
I think that the convention so far has been to add model related helper functions to the model classes themselves, see for example at the bottom of the project model. I'd rather keep that convention, easier to find code that way.